astropy/timeseries/periodograms/lombscargle/implementations/fastchi2_impl.py

```diff
@@ -18,6 +18,7 @@ def lombscargle_fastchi2(
     nterms=1,
     use_fft=True,
     trig_sum_kwds=None,
+    algorithm='lra',
```

Comment:
```suggestion
    *,
    algorithm="lra",
```
(note that I'm using double quotes for strings because that's what our formatter, ruff format, will enforce anyway)